protocol/communities_messenger_test.go

```diff
@@ -3494,6 +3471,78 @@ func (s *MessengerCommunitiesSuite) TestCommunityBanUserRequestToJoin() {
 	s.Require().ErrorContains(err, "can't request access")
 }
 
+func (s *MessengerCommunitiesSuite) TestCommunityMaxNumberOfMembers() {
+	john := s.newMessenger()
+	_, err := john.Start()
+	s.Require().NoError(err)
+
+	defer TearDownMessenger(&s.Suite, john)
+
+	community, _ := s.createCommunity()
+
+	communities.SetMaxNbMembers(2)
```

Comment:
I just met a problem today: if you change a global variable, it will remain changed in all subsequent tests within one run, so it's safer to return the old value at the end of the test run.